packages/protocol-kit/src/types/safeFactory.ts

```diff
@@ -24,7 +25,7 @@ export type SafeFactoryConfig = {
 
 export type SafeFactoryInitConfig = {
   provider: SafeProviderConfig['provider']
-  signer?: SafeProviderConfig['signer']
+  signer?: string | PasskeyArgType
```

Comment:
This is a regression, right?

Should be as it was

```suggestion
  signer?: SafeProviderConfig['signer']
```

As the `SafeProviderConfig['signer']` already includes the `PasskeyArgType`

Reply:
yes! fixed here:

```
https://github.com/safe-global/safe-core-sdk/pull/841/commits/801f8f237f9da2dc1a1adc7d4f844c79d46e5296
```